.github/workflows/ci.yml

```diff
@@ -4,8 +4,14 @@ on:
   - push
   - pull_request
 
+permissions:
+  contents: read
+
 jobs:
   test:
+    permissions:
+      checks: write  # for coverallsapp/github-action to create new checks
+      contents: read  # for actions/checkout to fetch code
```

Comment:
Isn't this inherited from the same permission defined at the root level? If so, can you please remove it?